cmd/auth/login.go

```diff
@@ -93,23 +91,18 @@ depends on the existing profiles you have set in your configuration file
 
 	cmd.RunE = func(cmd *cobra.Command, args []string) error {
 		ctx := cmd.Context()
+		profileName := cmd.Flag("profile").Value.String()
 
-		var profileName string
-		profileFlag := cmd.Flag("profile")
-		if profileFlag != nil && profileFlag.Value.String() != "" {
```

Comment:
This checks for `profileFlag != nil`. Is this command guaranteed to have the profile flag?

Reply:
Yeah, `--profile` is configured to be a root-level persistent flag in the CLI. 
https://github.com/databricks/cli/blob/48ff18e5fc6c9b5a6d9cebf08b019af1e6f9ada0/cmd/root/root.go#L42